apps/explorer/components/Contract/index.tsx

```diff
@@ -2,152 +2,162 @@
-          ? contract.proof_height.toLocaleString()
-          : '-',
-      },
-      {
-        label: 'proof time',
-        copyable: false,
-        value:
-          contract.proof_timestamp !== '0001-01-01T00:00:00Z'
-            ? humanDate(contract.proof_timestamp, {
-                dateStyle: 'medium',
-                timeStyle: 'short',
-              })
-            : '-',
+        value: contract.fileContract.windowStart.toLocaleString() || '-',
       },
+      // {
+      //   label: 'expiration time',
+      //   copyable: false,
+      //   value:
+      //     contract.expiration_timestamp !== '0001-01-01T00:00:00Z'
+      //       ? humanDate(contract.expiration_timestamp, {
+      //           dateStyle: 'medium',
+      //           timeStyle: 'short',
+      //         })
+      //       : '-',
+      // },
+      // {
+      //   label: 'proof height',
+      //   copyable: false,
+      //   value: contract.proof_height
+      //     ? contract.proof_height.toLocaleString()
+      //     : '-',
+      // },
+      // {
+      //   label: 'proof time',
+      //   copyable: false,
+      //   value:
+      //     contract.proof_timestamp !== '0001-01-01T00:00:00Z'
+      //       ? humanDate(contract.proof_timestamp, {
+      //           dateStyle: 'medium',
+      //           timeStyle: 'short',
+      //         })
+      //       : '-',
+      // },
       {
         label: 'proof deadline height',
         copyable: false,
-        value: contract.proof_deadline?.toLocaleString() || '-',
-      },
-      {
-        label: 'proof deadline time',
-        copyable: false,
-        value:
-          contract.proof_deadline_timestamp !== '0001-01-01T00:00:00Z'
-            ? humanDate(contract.proof_deadline_timestamp, {
-                dateStyle: 'medium',
-                timeStyle: 'short',
-              })
-            : '-',
-      },
-      {
-        label: 'payout height',
-        copyable: false,
-        value: contract.payout_height?.toLocaleString() || '-',
-      },
-      {
-        label: 'payout time',
-        copyable: false,
-        value:
-          contract.payout_timestamp !== '0001-01-01T00:00:00Z'
-            ? humanDate(contract.payout_timestamp, {
-                dateStyle: 'medium',
-                timeStyle: 'short',
-              })
-            : '-',
+        value: contract.fileContract.windowEnd.toLocaleString() || '-',
       },
+      // {
+      //   label: 'proof deadline time',
+      //   copyable: false,
+      //   value:
+      //     contract.proof_deadline_timestamp !== '0001-01-01T00:00:00Z'
+      //       ? humanDate(contract.proof_deadline_timestamp, {
+      //           dateStyle: 'medium',
+      //           timeStyle: 'short',
+      //         })
+      //       : '-',
+      // },
+      // {
+      //   label: 'payout height',
+      //   copyable: false,
+      //   value: contract.payout_height?.toLocaleString() || '-',
+      // },
```

Comment:
This is how we calculate payout height in renterd:
```
  const payoutHeight =
    c.resolutionHeight > 0
      ? c.resolutionHeight + 144
      : c.revision.windowEnd + 144
```
We just first need to confirm how to get resolution height.